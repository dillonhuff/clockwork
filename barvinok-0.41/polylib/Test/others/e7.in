# Haghighat and Polychronopoulos example
# 0 <= i <= N
# 3 <= j <= i
# j <= k <= 5
#------------
6 6
1  1  0  0  0  0
1 -1  0  0  1  0
1  0  1  0  0 -3
1  1 -1  0  0  0
1  0 -1  1  0  0
1  0  0 -1  0  5

1 3
1 1 0 

N
Haghighat and Polychronopoulos first example
