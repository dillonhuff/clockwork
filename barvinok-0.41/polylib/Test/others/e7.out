---------------------------------------
Domain :
         P  -5 >= 0
          1 >= 0

Vertices :
[ 5,  5,  5 ]
[ 3,  3,  5 ]
[ 3,  3,  3 ]
[ P,  5,  5 ]
[ P,  3,  5 ]
[ P,  3,  3 ]

Ehrhart Polynomial:
( 6 * P + -16 )

---------------------------------------
Domain :
         P  -3 >= 0
         - P + 5 >= 0

Vertices :
[ 3,  3,  5 ]
[ 3,  3,  3 ]
[ P,  P,  5 ]
[ P,  P,  P ]
[ P,  3,  5 ]
[ P,  3,  3 ]

Ehrhart Polynomial:
( 1/2 * P^2 + 3/2 * P + -6 )

